src/firmware/main/app/world/wheel.h

```diff
@@ -30,3 +61,24 @@ void app_wheel_destroy(Wheel_t* wheel);
  * @param force_in_newtons The force to apply to the wheel, in newtons
  */
 void app_wheel_applyForce(Wheel_t* wheel, float force_in_newtons);
+
+/**
+ * Get the speed of the given wheel in RPM
+ * @param wheel The wheel to get the speed for
+ * @return The speed of the given wheel in RPM
+ */
+float app_wheel_getWheelSpeedRPM(Wheel_t* wheel);
```

Comment:
Do we need a convention for wheel direction? aka what positive and negative values of speed mean in terms of which way the wheel is rotating? This context is important for anyone using this function.

It's likely the old firmware already had / followed one, we should just double check and make sure we still obey it, and that it's documented.

Reply:
I don't trust my ability to go through the firmware and confidently say what direction this is. Let's leave this comment open and we can check with the final hardware test.